datasets/diplomacy_detection/README.md

```diff
@@ -0,0 +1,100 @@
+---
+YAML tags:
+annotations_creators: [no-annotation]
+language_creators: [found]
+languages: [en]
+licenses: [unknown]
+multilinguality: [monolingual]
+size_categories: [n<1K]
+source_datasets: [original]
+task_categories: [text-classification]
+task_ids: [intent-classification]
+---
+
+# Dataset Card for HateOffensive
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
```

Comment:
Some sections are missing.
It would be great if you could provide them with relevant information !
Please at least add them with `[More information needed]` as a body 🤗